src/site/content/en/lighthouse-performance/third-party-summary/index.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: Reduce the impact of third-party code
+description: |
+  Learn how third-party code, like advertising networks and analytics services, affects page load performance,
+  and how you can optimize third-party code.
+web_lighthouse:
+  - third-party-summary
+---
+
+To add an advertising network, social media button, A/B test, or analytics service to your page,
+you usually need to add a third-party script to your HTML. These [third-party scripts can
+significantly affect your page load performance](/third-party-javascript/).
+
+## How the Lighthouse audit for third-party code fails
+
+Lighthouse flags pages that have third-party code which blocks the [main thread] for 250&nbsp;ms or
+longer:
+
+<figure class="w-figure">
+  <img class="w-screenshot w-screenshot--filled" src="third-party-summary.png" 
+       alt="A screenshot of the 'Reduce the impact of third-party code' audit.">
+  <figcaption class="w-figcaption">
+    <b>Reduce the impact of third-party code</b>
+  </figcaption>
+</figure>
+
+A third-party script is any script hosted on a domain that's different than the domain of the URL
+that you audited with Lighthouse. As the page loads, Lighthouse calculates how long each of the
+third-party scripts blocks the main thread. If the total blocking time is greater than 250&nbsp;ms
+the audit fails.
```

Comment:
The last 2 sentences are now a bit redundant with the 1st sentence of the section, but the last 2 clarify that the 250 ms threshold is the combined blocking time of all 3rd-party code.

Reply:
LGTM. This is more or less how all the failure sections work: first sentence gives a summary; the rest of the section explicates.